src/Compilers/Core/Portable/Text/LargeText.cs

```diff
@@ -286,5 +251,10 @@ private int[] ParseLineStarts()
             arrayBuilder.Add(position);
             return arrayBuilder.ToArrayAndFree();
         }
+
+        private readonly struct ChunkHelper : ChunkedTextUtilities.IChunkHelper<char[]>
+        {
+            public int GetLength(char[] chunk) => chunk.Length;
+        }
```

Comment:
do we need tehse interfaces?  or can this just be a static lambda passed into the caller?

Reply:
No they don't, I can change it to a lambda if that's what you prefer. I just thought this was a little cleaner and it prevents us from having to pass in a delegate as an argument.

Reply:
actually, keep like this.  this should allow for better codegen through generic specialization.

Reply:
That's what I was thinking as well.